include/extractor/guidance/turn_handler.hpp

```diff
@@ -68,8 +85,7 @@ class TurnHandler : public IntersectionHandler
     handleDistinctConflict(const EdgeID via_edge, ConnectedRoad &left, ConnectedRoad &right) const;
 
     // Classification
-    std::pair<std::size_t, std::size_t> findFork(const EdgeID via_edge,
-                                                 const Intersection &intersection) const;
+    boost::optional<Fork> findFork(const EdgeID via_edge, Intersection &intersection) const;
```

Comment:
could you maybe add documentation on what the difference between `findFork` and `findLeftmost...` versions are?